client/post-editor/editor-title/index.jsx

```diff
@@ -4,27 +4,33 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
 import omit from 'lodash/omit';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import PostActions from 'lib/posts/actions';
 import PostUtils from 'lib/posts/utils';
 import SiteUtils from 'lib/site/utils';
 import EditorPermalink from 'post-editor/editor-permalink';
 import TrackInputChanges from 'components/track-input-changes';
 import FormTextInput from 'components/forms/form-text-input';
 import { isMobile } from 'lib/viewport';
 import * as stats from 'lib/posts/stats';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { editPost } from 'state/posts/actions';
+import { getEditedPostValue } from 'state/posts/selectors';
 
-export default React.createClass( {
-	displayName: 'EditorTitle',
-
+const EditorTitle = React.createClass( {
 	propTypes: {
 		post: PropTypes.object,
 		site: PropTypes.object,
+		siteId: PropTypes.number,
```

Comment:
With this new prop, should we tidy up line 57 that is still using `this.props.site.ID`